### DISCUSSION TITLE
got this error trying to install MagiskOnWSALocal during image mount

### DISCUSSION THREAD

**Discussion by Plaguewave7** [LSPosed/MagiskOnWSALocal#692] [Q&A]
I got this error trying to do an install, the previous issue I had was resolved, but now, it's errored out while trying to mount the image

![image](https://github.com/LSPosed/MagiskOnWSALocal/assets/71691723/cf734be8-8b3b-4156-9072-67c64ef846b3)

**Comment by sn-o-w** [comment]
You need this commit: https://github.com/LSPosed/MagiskOnWSALocal/pull/665
